connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroCatalystDataConversionSuite.scala

```diff
@@ -360,4 +362,27 @@ class AvroCatalystDataConversionSuite extends SparkFunSuite
       None,
       new OrderedFilters(Seq(Not(EqualTo("Age", 39))), sqlSchema))
   }
+
+  test("AvroDeserializer with binary type") {
+    val jsonFormatSchema =
+      """
+        |{ "type": "record",
+        |  "name": "record",
+        |  "fields" : [{
+        |    "name": "a",
+        |    "type": {
+        |      "type": "bytes"
+        |    }
```

Comment:
Why is there a nested type here instead of just `"type": "bytes"` at the top-level?

Reply:
This is not necessary and it has been removed.